packages/components/icons/src/icons/Check.tsx

```diff
@@ -11,7 +11,7 @@ export const Check = ({
     xmlns="http://www.w3.org/2000/svg"
     {...{ ...(title && { 'data-title': title }), fill, stroke, ...props }}
   >
-    ${title === undefined ? undefined : <title>{title}</title>}
+    {title === undefined ? undefined : <title>{title}</title>}
```

Comment:
```suggestion
    {title ? <title>{title}</title> : undefined}
```

Reply:
falsy values are awared

Reply:
I'm not sure I understand?
We don't want falsy values to be rendered, right?